tests/commands/logs.ts

```diff
@@ -12,24 +12,29 @@ import {
   setupWithoutGCPProject,
 } from '../functions';
 
+import {
+  LOG,
+} from '../../src/utils';
+
 describe('Test clasp logs function', () => {
```

Comment:
Change the description of this test, since it's not actually testing the logging function, it's also confirming the setup.

Reply:
Ok, I will change the description and write additional tests for the rest of the logging function.